packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -197,14 +197,18 @@ export default class PciInstancesAddController {
 
     this.onFlexChange(false);
 
-    if (this.model.image.type !== 'linux') {
+    if (!this.isLinuxImageType()) {
       this.model.sshKey = null;
       this.instance.sshKeyId = null;
     } else {
       this.instance.sshKeyId = get(this.model.sshKey, 'id');
     }
   }
 
+  isLinuxImageType() {
+    return this.model.image.type.indexOf('linux') !== -1;
```

Comment:
**suggestion:** `string.prototype.includes` return a boolean. So we can use it to make more readable.
```suggestion
    return !this.model.image.type.includes('linux');
```

Reply:
It's applied